examples/sugarscape_g1mt/sugarscape_g1mt/server.py

```diff
@@ -28,15 +29,25 @@ def SsAgent_portrayal(agent):
         portrayal["w"] = 1
         portrayal["h"] = 1
 
+    elif type(agent) is Spice:
+        if agent.amount != 0:
+            portrayal["Color"] = spice_dic[agent.amount]
+        else:
+            portrayal["Color"] = "#D6F5D6"
```

Comment:
What color is this?

Reply:
I just checked; it's a nice green color. However, both sugar and spice seem to have the same color.

Reply:
I made spice a "wheat" color just for test purposes. Are you saying you want them to have the same color or they do have the same color? It should look like....

![image](https://user-images.githubusercontent.com/22775448/178694705-a5882884-1d36-49dc-8d7e-451586b516e7.png)

My intent is to keep them different colors until I code the larger color (i.e. more sugar or more spice) taking precedence. .

Reply:
Oh, I wasn't aware that the color is different. Yes, definitely different color.

Reply:
@tpike3 that looks good to me.